conf.d/python.d/ipfs.conf

```diff
@@ -39,6 +39,10 @@
 # This feature is disabled by default.
 # autodetection_retry: 0
 
+# Disable/Enable IPFS pin api
+# Currently defaults to disabled due to IPFS Bug(https://github.com/ipfs/go-ipfs/issues/3874) resulting in very high CPU Usage
+# pinapi: False
+
```

Comment:
Move it to the `Additionally to the above, ipfs also supports the following` section.